packages/core/src/components/index.ts

```diff
@@ -48,11 +48,8 @@ export * from "./non-ideal-state/nonIdealState";
 export * from "./overlay/overlay";
 export * from "./text/text";
 export * from "./popover/popover";
-export * from "./popover2/popover2";
-export * from "./popover/svgPopover";
 export * from "./portal/portal";
 export * from "./progress/progressBar";
-export * from "./tooltip/svgTooltip";
```

Comment:
@adidahiya @giladgray React 16 was causing `SVGTooltip` and `SVGPopover` tests to fail, because they were trying to render `<g>` tags outside of an `<svg>`. Since these components are such thin wrappers around the core `Tooltip` and `Popover`, respectively, I opted just to delete them and consider adding documentation about working with these components in SVG land.

Does that sound okay?

Reply:
code search reveals a very small number (~5) of `SVGPopover` usages, and it's possible some of those are stale. I hate to say it, but I think we'll want to at least preserve support for this use case, which may require some work to be able to customize elements.